app/helpers/sufia/sufia_helper_behavior.rb

```diff
@@ -104,7 +104,9 @@ def link_to_field(name, value, label = nil, facet_hash = {})
     # @see #index_field_link params
     # @return [Date]
     def human_readable_date(options)
-      Date.parse(options[:value]).to_formatted_s(:standard)
```

Comment:
@mtribone OK. Could you revert the change and then post the full traceback, so we can look to reproduce and investigate?